swap/swap_test.go

```diff
@@ -832,55 +823,43 @@ func testDeploy(ctx context.Context, backend cswap.Backend, swap *Swap) (err err
 	return err
 }
 
-// TestSaveAndLoadLastReceivedCheque tests if a saved last received cheque can be loaded again later using the swap functions
-func TestSaveAndLoadLastReceivedCheque(t *testing.T) {
-	swap, clean := newTestSwap(t, ownerKey)
-	defer clean()
-
-	testPeer := NewPeer(newDummyPeer().Peer, swap, common.Address{}, common.Address{})
-	testCheque := newTestCheque()
-
-	if err := swap.saveLastReceivedCheque(testPeer, testCheque); err != nil {
-		t.Fatalf("Error while saving: %s", err.Error())
-	}
-
-	returnedCheque := swap.loadLastReceivedCheque(testPeer)
-
-	if returnedCheque == nil {
-		t.Fatalf("Could not find saved cheque")
-	}
-
-	if !returnedCheque.Equal(testCheque) {
-		t.Fatalf("Returned cheque was different")
-	}
-}
-
 // newTestSwapAndPeer is a helper function to create a swap and a peer instance that fit together
 // the owner of this swap is the beneficiaryAddress
 // hence the owner of this swap would sign cheques with beneficiaryKey and receive cheques from ownerKey (or another party) which is NOT the owner of this swap
 func newTestSwapAndPeer(t *testing.T, key *ecdsa.PrivateKey) (*Swap, *Peer, func()) {
 	swap, clean := newTestSwap(t, key)
 	// owner address is the beneficiary (counterparty) for the peer
 	// that's because we expect cheques we receive to be signed by the address we would issue cheques to
-	peer := NewPeer(newDummyPeer().Peer, swap, ownerAddress, testChequeContract)
+	peer := swap.newPeer(newDummyPeer().Peer, ownerAddress, testChequeContract)
 	// we need to adjust the owner address on swap because we will issue cheques to beneficiaryAddress
 	swap.owner.address = beneficiaryAddress
 	return swap, peer, clean
 }
 
-// TestPeerSaveAndLoadLastReceivedCheque tests if a saved last received cheque can be loaded again later using the peer functions
-func TestPeerSaveAndLoadLastReceivedCheque(t *testing.T) {
+// TestPeerSaveAndLoadgetLastReceivedCheque tests if a saved last received cheque can be loaded again later using the peer functions
+func TestPeerSaveAndLoadgetLastReceivedCheque(t *testing.T) {
```

Comment:
I understand the good intention, but please use CamelCase consistently

Reply:
Looks like an unintentional consequence of a global replace. Fixed.